src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head class="page">
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
+  </head>
+  <body class="page">
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="image APPLE iMac Monoblock"
+      >
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4 card__rating-stars">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
```

Comment:
Split your blocks by new line it is a good practice and I like it in your html), BUT in this case I would recommend don't do this, because those tags are one-line and easy to read even when they are placed line by line